queries/qm/query_mods.go

```diff
@@ -2,8 +2,21 @@ package qm
 
 import "github.com/volatiletech/sqlboiler/queries"
 
-// QueryMod to modify the query object
-type QueryMod func(q *queries.Query)
+// QueryMod modifies a query object.
+type QueryMod interface {
+	Mod(q *queries.Query)
```

Comment:
Should this be named `Apply`? Not sure, I'm on the fence about it. I'll let you make the final call.

Reply:
Done.

I like `Apply` one and also is more consistent with the array `Apply`.